main.go

```diff
@@ -92,6 +98,30 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "ServiceAccount")
 		os.Exit(1)
 	}
+
+	ctx := context.Background()
+	gkehubClient, err := gkehub.NewGkeHubMembershipClient(ctx)
```

Comment:
Using this makes us a different error handling function that we use in other components/operators where we use the REST client. Let's use the REST client here too.